addons/account/models/account_move_line.py

```diff
@@ -2903,6 +2924,24 @@ def remove_move_reconcile(self):
         """ Undo a reconciliation """
         (self.matched_debit_ids + self.matched_credit_ids).unlink()
 
+    def _reconcile_marked(self):
+        temp_numbers = list({
+            line.matching_number
+            for line in self
+            if line.matching_number and line.matching_number.startswith('I')
+        })
+        if temp_numbers:
+            for _matching_number, account, lines in self._read_group(
+                domain=[('matching_number', 'in', temp_numbers)],
+                groupby=['matching_number', 'account_id'],
+                aggregates=['id:recordset'],
+            ):
+                if all(move.state == 'posted' for move in lines.move_id):
+                    if not account.reconcile:
+                        _logger.info("%s has reconciled lines, changing the config", account.display_name)
+                        account.reconcile = True
```

Comment:
mmhh are you sure we want to do that? setting the reconcile to True triggers the computation of amount_residual for all lines having that account. We could raise an error... :thinking: 

or maybe it's just not at its rightful place? not sure it would be better in the post(), though...

Reply:
If it was imported as reconciled, we want it reconciled in Odoo too
That's how it always worked for FEC and Winbooks imports

Reply:
ok fine for me :)